### PR TITLE
MINOR: Add a unit test for GroupMetadataManager#offsetExpiredSensor

### PR DESCRIPTION
**Pull request by stanislavkozlovski**
This patch is a continuation of b7cab2d66f017464d3242525643a2afe2c28e924 which fixed a bug in the recording of the sensor but did not follow the policy of adding a test for said bugfix

### CODE REVIEW

**Comment by stanislavkozlovski** [comment]
cc @omkreddy @dajac @ijuma

File: core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala [hunk 1]
```diff
@@ -791,52 +797,52 @@ class GroupMetadataManager(brokerId: Int,
...
       }
 
-    val offsetsPartition = partitionFor(groupId)
-    val appendPartition = new TopicPartition(Topic.GROUP_METADATA_TOPIC_NAME, offsetsPartition)
-    getMagic(offsetsPartition) match {
-      case Some(magicValue) =>
-        // We always use CREATE_TIME, like the producer. The conversion to LOG_APPEND_TIME (if necessary) happens automatically.
-        val timestampType = TimestampType.CREATE_TIME
-        val timestamp = time.milliseconds()
+      val offsetsPartition = partitionFor(groupId)
```
Comment (stanislavkozlovski) [inline core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala:800]: I fixed the indentation here while exploring mocks in a different way of testing this feature. I think it'd be good to commit this as it is confusing to read

**Comment by stanislavkozlovski** [comment]
cc @hachikuji

**Comment by stanislavkozlovski** [comment]
cc @dajac @ijuma

File: core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala (hunk 1 above)
Comment (chia7712) [reply core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala:800]: make sense to me. Could you fix the conflicting files?

Comment (stanislavkozlovski) [reply core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala:800]: Will do!

Comment (chia7712) [reply core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala:800]: @stanislavkozlovski Any update? This part is worth being a separate PR :)

**Pull request closed without merging**